streams.json

```diff
@@ -1851,48 +2357,56 @@
               "rhusar@redhat.com"
           ],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/jgroups-extras/jgroups-azure",
+          "repository_url": "https://github.com/jgroups-extras/jgroups-azure.git",
           "codebase": "master",
           "tag": "1.1.0.Final",
+          "group_id": "org.jgroups",
+          "artifact_id": "jgroups-azure-parent",
           "version": "1.1.0.Final",
-          "gav": "org.jgroups",
+          "gav": "jgroups-azure-parent:org.jgroups:1.1.0.Final",
           "comment": ""
         },
         {
           "component_name": "jhy-jsoup",
           "contacts": [],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/jhy/jsoup",
+          "repository_url": "https://github.com/jhy/jsoup.git",
           "codebase": "master",
           "tag": "jsoup-1.8.3",
+          "group_id": "org.jsoup",
+          "artifact_id": "jsoup",
           "version": "1.8.3",
-          "gav": "org.jsoup",
-          "comment": " There is no maintenance branch for 1.8, master is at 1.10"
+          "gav": "jsoup:org.jsoup:1.8.3",
+          "comment": "TAG suspicious. TAG in pom: jsoup-1.8.3a, TAG in mapping tool: jsoup-1.8.3.  There is no maintenance branch for 1.8, master is at 1.10"
         },
         {
           "component_name": "JIPIJAPA",
```

Comment:
It looks like an obsolete component. It seems jipijapa is a submodule instead of a separated component.